src/DDM.jl

```diff
@@ -125,230 +226,493 @@ function _large_time_pdf(u::T, z::T, K::Int) where {T<:Real}
+#                         ser = -((α*a2)/((1 - 2*choice)*gk[m]*π*.01))*sinh(zzz*(1 - 2*x)*gk[m]/.01)/
+#                               (sinh((1 - 2*choice)*gk[m]*α/.01)^2) +
+#                               (zzz*a2)/((1 - 2*choice)*gk[m]*π*.01)*cosh((α - zzz)*(1 - 2*choice)*gk[m]/.01)/
+#                               sinh((1 - 2*choice)*gk[m]*α/.01)
+#                         sum_hist = zeros(3)
+#                         for v in 1:v_max
+#                             sum_hist = circshift(sum_hist, 1)
+#                             sifa = π*v/α
+#                             denom = (gk[m]*gk[m]*100 + (π*v)*(π*v)/(a2*100))
+#                             sum_hist[3] = sum_hist[2] + v*sin(sifa*zzz)*exp(-.5*denom*(rt - lower_t) - 2*log(denom))
+#                             if abs(sum_hist[1] - sum_hist[2]) < Δ && abs(sum_hist[2] - sum_hist[3]) < Δ && sum_hist[3] > 0
+#                                 break
+#                             end
+#                         end
+#                         sum_z += .5*w_g[i]*(ser - 4*sum_hist[3])*(π/100)/(a2*st)*exp((2*choice - 1)*zzz*gk[m]*100)
+#                     end
+#                 else
+#                     sum_hist = zeros(3)
+#                     su = -(Z_U*Z_U)/(12*a2) + (Z_U*Z_U*Z_U)/(12*α*a2) - (Z_U*Z_U*Z_U*Z_U)/(48*a2*a2)
+#                     sl = -(Z_L*Z_L)/(12*a2) + (Z_L*Z_L*Z_L)/(12*α*a2) - (Z_L*Z_L*Z_L*Z_L)/(48*a2*a2)
+#                     for v in 1:v_max
+#                         sum_hist = circshift(sum_hist, 1)
+#                         sifa = π*v/α
+#                         denom = (π*v)*(π*v)/(a2*100)
+#                         sum_hist[3] = sum_hist[2] + 1/(π*π*π*π*v*v*v*v)*(cos(sifa*Z_L) - cos(sifa*Z_U))*
+#                                       exp(-.5*denom*(rt - lower_t))
+#                         if abs(sum_hist[1] - sum_hist[2]) < Δ && abs(sum_hist[2] - sum_hist[3]) < Δ && sum_hist[3] > 0
+#                             break
+#                         end
+#                     end
+#                     sum_z = 400*a2*α*(sl - su - sum_hist[3])/(st*sz)
+#                 end
+#                 sum_ν += sum_z*w_gh[m]
+#             end
+#             Fnew = (p0*(1 - choice) + p1*choice) - sum_ν
+#         end
+#     elseif rt - τ + st/2 <= min_rt # is t lower than lower boundary Ter distr?
+#         Fnew = 0
+#     end
+    
+#     Fnew = Fnew > Δ ? Fnew : 0
+
+#     return    Fnew
+
+# end
+
 """
     rand(dist::DDM)
 
-Generate a random rt for the Diffusion Decision Model (negative coding)
+Generate a random choice and rt for the Diffusion Decision Model
 
 # Arguments
-- `dist`: model object for the Diffusion Decision Model. 
+- `dist`: model object for Diffusion Decision Model. 
+    
+method simulating the diffusion process: 
+_rand_rejection uses Tuerlinckx et al., 2001 rejection-based method for the general wiener process
+_rand_stochastic uses the stochastic Euler method to simulate the stochastic differential equation
```

Comment:
What is the status with these different  methods for `rand`? If you are going to implement them later, can you remove the doc string  comment for `_rand_stochastic` for the time being?